server/package.json

```diff
@@ -3,7 +3,7 @@
   "version": "0.0.0",
   "private": true,
   "scripts": {
-    "start": "node --inspect=0.0.0.0 build/index.js",
+    "start": "tsc --project ./ && node --inspect=0.0.0.0 build/index.js",
```

Comment:
@Abeilles14 does this fix our typescript compiling issues?

Reply:
to fix it I just needed to rebuild with `npm run build` i think, but it can't hurt to include this